npc/re/instances/SaraMemory.txt

```diff
@@ -0,0 +1,1638 @@
+//===== rAthena Script =======================================
+//= Sara's Memories
+//===== By: ==================================================
+//= Chilly
+//===== Current Version: =====================================
+//= 1.0
+//===== Compatible With: =====================================
+//= rAthena Project
+//===== Description: =========================================
+//= Sara's Memories Instance in the Dimensional Gap.
+//===== Additional Comments: =================================
+//= 1.0 First Release
+//============================================================
+
```

Comment:
can u change the script header follow this? https://rathena.org/board/topic/100232-script-header/